.github/actions/integration-tests/run-ci-stage1

```diff
@@ -655,11 +669,25 @@ for scenario in $(echo "${CI_SCENARIOS}" | sed -e "s/,/ /g"); do
     run_cmd "crucible unarchive /var/lib/crucible/archive/${scenario}*"
 done
 
-run_cmd "crucible repo config update name=external-userenvs checkout-mode=locked" "force"
+case "${CI_RELEASE}" in
+    "2024.4"|"2025.1"|"2025.2"|"2025.3")
```

Comment:
can't be this in the repos.json or some other config file ? maintaining these lists will be a nightmare looks like...